packages/k10/package.yaml

```diff
@@ -0,0 +1,2 @@
+url: https://charts.kasten.io/k10-4.5.6.tgz
+version: 4.5.6
```

Comment:
this provides the chart URL to use and prepare the charts and other related files.